src/main/java/org/folio/circulation/resources/AddInfoResource.java

```diff
@@ -91,25 +92,35 @@ private Result<AddInfoRequest> createAddInfoRequest(RoutingContext routingContex
   }
 
   private boolean staffOrPatronInfoProvidedInRequest(JsonObject requestBody) {
+    log.debug("staffOrPatronInfoProvidedInRequest:: parameters requestBody: {}", requestBody);
     String action = requestBody.getString(ACTION);
     return StringUtils.isNotBlank (action)
       && StringUtils.isNotBlank(requestBody.getString(ACTION_COMMENT))
       && (action.equals(PATRON_INFO_ADDED.getValue()) || action.equals(STAFF_INFO_ADDED.getValue()));
   }
 
-  CompletableFuture<Result<Loan>> getExistingLoan(LoanRepository loanRepository, AddInfoRequest addInfoRequest) {
+  CompletableFuture<Result<Loan>> getExistingLoan(LoanRepository loanRepository,
+    AddInfoRequest addInfoRequest) {
+
     return loanRepository.getById(addInfoRequest.getLoanId());
   }
 
-  private Result<LoanAndRelatedRecords> addPatronOrStaffInfo(Result<LoanAndRelatedRecords> loanResult,
-                                                             AddInfoRequest request) {
-    return loanResult.map(l -> addPatronOrStaffInfo(l, request.getAction(), request.getActionComment()));
+  private Result<LoanAndRelatedRecords> addPatronOrStaffInfo(
+    Result<LoanAndRelatedRecords> loanResult, AddInfoRequest request) {
+
+    return loanResult.map(l -> addPatronOrStaffInfo(l, request.getAction(),
+      request.getActionComment()));
   }
 
   private LoanAndRelatedRecords addPatronOrStaffInfo(LoanAndRelatedRecords loanAndRelatedRecords,
-                                                     String action, String actionComment) {
+    String action, String actionComment) {
+
+    log.debug("addPatronOrStaffInfo:: parameters loanAndRelatedRecords: {}, action: {}, " +
+      "actionComment: {}", loanAndRelatedRecords, action, actionComment);
```

Comment:
logging `loanAndRelatedRecords` converts a loan JSON-representation into a String. Are we sure we want the whole loan in the logs? Also, it's a rather costly operation, so we better use Supplier or check if debug-level is enabled.